WISE-Community/WISE
# Issue 884 share filter

To test:
1. Log in as user with admin permissions.
2. Find a run (or project) by ID
3. Click on "share with other teachers" in the listing
4. Type your own username. It *should* show up.
5. Type in the run owner's username. It *should not* show up.
